# HG changeset patch
# User Takuro Ashie <ashie@clear-code.com>
# Date 1536568628 -32400
#      Mon Sep 10 17:37:08 2018 +0900
# Node ID c42d32e54ef0b80cc9be36193d887efddb75476d
# Parent  119260545b752ef1c7ca54afa7c94a3b090b8199
Don't test OMX_UseEGLImage()

Because It causes redundant warning messages on RZ/G1.

diff --git a/dom/media/platforms/omx/PureOmxPlatformLayer.cpp b/dom/media/platforms/omx/PureOmxPlatformLayer.cpp
--- a/dom/media/platforms/omx/PureOmxPlatformLayer.cpp
+++ b/dom/media/platforms/omx/PureOmxPlatformLayer.cpp
@@ -25,20 +25,22 @@ namespace mozilla {
 PureOmxBufferData::PureOmxBufferData(const PureOmxPlatformLayer& aPlatformLayer,
                                      const OMX_PARAM_PORTDEFINITIONTYPE& aPortDef)
   : BufferData(nullptr)
   , mPlatformLayer(aPlatformLayer)
   , mPortDef(aPortDef)
 {
   LOG_BUF("");
 
+#if 0
   if (ShouldUseEGLImage()) {
     // TODO
     LOG_BUF("OMX_UseEGLImage() seems available but using it isn't implemented yet.");
   }
+#endif
 
   OMX_ERRORTYPE err;
   err = OMX_AllocateBuffer(mPlatformLayer.GetComponent(),
                            &mBuffer,
                            mPortDef.nPortIndex,
                            this,
                            mPortDef.nBufferSize);
   if (err != OMX_ErrorNone) {
